bot/utils/members.py

```diff
@@ -0,0 +1,24 @@
+import logging
+import typing as t
+
+import discord
+
+log = logging.getLogger(__name__)
+
+
+async def get_or_fetch_member(guild: discord.Guild, member_id: int) -> t.Optional[discord.Member]:
+    """
+    Attempts to get a member from cache, on failure tries to fetch from the API.
+
+    Returns None if member is not part of the guild.
+    """
```

Comment:
The [pydis style guide recommends imperative mood](https://pythondiscord.com/pages/guides/pydis-guides/contributing/style-guide/#mood). :smile:

Reply:
Updated the docstring